src/main/kotlin/io/libp2p/pubsub/gossip/GossipScore.kt

```diff
@@ -224,6 +224,7 @@ class GossipScore(
         }
     }
 
+    @Suppress("UNUSED_PARAMETER")
     fun notifyUnseenMessage(peer: P2PService.PeerHandler, msg: PubsubMessage) {
```

Comment:
Just to double check, this empty function is correct right? I'm guessing that unseen messages have no effect on scores and therefore are not processed.

Reply:
Yes I believe that's correct. We should get a notifyUnseenValidMessage or notifyUnseenInvalidMessage call once validation completes which is where scoring is updated.